models/block.go

```diff
@@ -49,3 +47,9 @@ func DecodeBlock(event cadence.Event) (*types.Block, error) {
 		TransactionHashes: hashes,
 	}, nil
 }
+
+// IsBlockExecutedEvent checks whether event contains block executed data.
+func IsBlockExecutedEvent(event cadence.Event) bool {
+	blockExecutedType := (types.EVMLocation{}).TypeID(nil, string(types.EventTypeBlockExecuted))
```

Comment:
I believe that having `blockExecutedType` as a constant was better in terms of object allocations. Right now, for every single decoding of block event, a new `blockExecutedType` will be allocated.